### DISCUSSION TITLE
Vercel outage? 😱

### DISCUSSION THREAD

**Discussion by Omid-Mirzaee-Yazdi** [vercel/community#668] [Help] [answered]
I started having issues with the deploy hook which doesn't deploy my pr anymore and I can't access the dashboard it gives 502 status. but it shows all services are operational on https://www.vercel-status.com/

**Comment by Omid-Mirzaee-Yazdi** [comment]
Update: I see a partial outage on the status page but I am having issues with deploy hook from GitHub as well

**Reply by k11kirky** [reply]
same. Functions has since recovered, but we are seeing issues with the live site and cannot access dashboard

**Comment by leerob** [comment, marked as answer]
[Please see the incident](https://www.vercel-status.com/incidents/hfbqmbc1jgyx), where you can view the status and subscribe to updates. Thank you.

**Reply by Omid-Mirzaee-Yazdi** [reply]
yes but thats not accurate, I am having issues with few other services that is marked as operational

**Reply by leerob** [reply]
Incident has been updated 🙏
